admin/jenkins.py

```diff
@@ -12,11 +12,7 @@
 import sys
 import os
 import subprocess
-try:
```

Comment:
Do we know that this was only for python 2.6? (i.e., is the comment correct?)

Reply:
I think so.